en/devtools-usage.md

```diff
@@ -158,9 +158,9 @@ class TestController extends Controller
 
 <a name='database-settings'></a>
 ## Preparing Database Settings
-When a project is generated using developer tools. A configuration file can be found in `app/config/config.php`. To generate models or scaffold, you will need to change the settings used to connect to your database.
+When a project is generated using developer tools. A configuration file can be found in `app/config/config.ini`. To generate models or scaffold, you will need to change the settings used to connect to your database.
```

Comment:
@Ultimater Can you please add notice e.g. 

> Note: In future releases this file may named config.php. this is not a fundamental principle - the file name, just take in mind that you'll need tune up application using a configuration file

I'm not sure about exact notice form. Just an idea

Reply:
Noticing multiple things with the devtools config file that really should have its own dedication page for explaining what options are available, what makes a difference, what doesn't, etc.  
Webtools should also do a better job of explaining what went wrong when trying to locate the controllers directory or the models directory etc. And tell the end-user where it's looking, or whether it was never configuration in the first place.